spec/fixtures/rails5_users_app/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    image: rails-app:${RUBY_VERSION}
+    image: rails5-app:${RUBY_VERSION}
```

Comment:
@apotterri this actually fixes a bug in which the same Docker image (`rails-app`) was being used for both Rails5 and Rails6 testing. Which ever image was built last would be use by both test suites. So with this change, now we are reliably testing both Rails 5 and 6.